thrust/testing/scan.cu

```diff
@@ -74,6 +74,13 @@ void TestScanSimple()
   ASSERT_EQUAL(input, input_copy);
   ASSERT_EQUAL(output, result);
 
+  // inclusive scan with init and op
+  iter   = thrust::inclusive_scan(input.begin(), input.end(), output.begin(), T(3), thrust::plus<T>());
```

Comment:
Important: AFAIK, plus is the default operator used within inclusive scan. Consider taking a e.g `thrust::times` to ensure that we actually use the passed in operator